authortechnical.Rmd

```diff
@@ -391,7 +391,32 @@ Value 21 | Value 22       | Value 23
 Value 31 | Value 32       | Value 33
 {.table .table-responsive .table-striped}
 ```
+  
+### Math
+  
+If you want to use mathematical formula,
+  
+* install the package [katex](https://docs.ropensci.org/katex/).
+* add `math: true` to your post YAML.
+* for all math you'll need to double the backslashes. All "\" become "\\".
```

Comment:
Also, when I preview this locally, there are no backslashes...? Locally I had to add more to make this work
```suggestion
* for all math you'll need to double the backslashes, meaning all "\\\" become "\\\\\".
```